core/test/object-manipulator.ts

```diff
@@ -121,4 +125,31 @@ definitions:
       assert.strictEqual(resultObject.paths['/api/circular'].get.description, 'FUN TIME');
     }
   }
+
+  @test async 'skip'() {
```

Comment:
Was this meant to be skipped?

Reply:
Yeah, `skip` is not best name. What about `skip-transform-failure`?